packages/api/src/controllers/region.js

```diff
@@ -0,0 +1,63 @@
+import Router from 'express/lib/router'
+import { authMiddleware } from '../middleware'
+import { db } from '../store'
+
+const app = Router()
+
+app.get('/', authMiddleware({ admin: true }), async (req, res, next) => {
+  const [regions, cursor] = await db.region.find()
+  if (req.query.grouped) {
+    return res.json(regions)
+  }
+
+  const flatOrchList = []
+  regions.forEach((region) => {
+    region.orchestrators.forEach((orch) => {
+      orch.region = region.region
+      flatOrchList.push(orch)
+    })
+  })
+
+  return res.json(flatOrchList)
+})
+
+app.get('/:region', authMiddleware({ admin: true }), async (req, res, next) => {
+  const region = await db.region.get(req.params.region)
```

Comment:
Need to account for the 404 case here, or else we'll return a 200 with `null` as the result. (This changed a while back - back in the day a missing response threw a 404 error automatically.)